templates/account.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% block body %}
+<h1>Create an Account:</h1>
+<form action="/user/add" method="POST">
+    <font color='red' id='msg'></font><br>
+    <label>Username: </label><input type="text" name="handle" class="form-control"/><br/>
+    <label>Password: </label><input type="password" name="password" class="form-control"/><br/>
+    <input type="submit" class="btn btn-block btn-warning" value="Submit"><br/>
+</form>
+<script>
+var state = '{{ request.args.get("success") }}';
+if(state == 'create0'){
+    id('msg').innerHTML = 'You cannot create an account with this username.';
+}
```

Comment:
I wasn't explicit about this, when I said:

> Flash error message should display if username already exists in db

I was thinking of Flask's `flash` functionality. Please read this and consider implementing that way. (If you want to do it using JS, tell me why you think that is better.